### ISSUE TITLE
Cannot import lxml

### ISSUE THREAD

**Issue by mrmusicman** [brython-dev/brython#2127]
I am trying to build a webapp with brython and lxml, xml.dom.minidom, and bs4. i see a lot of errors in the console that have to do with error 404s when finding module .py files, but i do see that it found some of them somewhere else. there is still an issue I am getting with brython that I do not have with python

```
File "http://.../Lib/site-packages/bs4/builder/_lxml.py", line 193
    if len(markup) > 0 and markup[0] == u'\N{BYTE ORDER MARK}':
                                        ^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: (unicode error) unknown Unicode character name
```

I can run the same code in normal Python, meaning that there is some discrepancy between brython and python that needs to be fixed.

p.s. - A discord server to discuss brython would be useful for asking shorter questions that can be answered quickly

**Comment by mrmusicman** [comment]
For now, the only fix for the above error is to comment out those lines in _lxml.py, which must be done before building the javascript.

Another issue:
```
Traceback (most recent call last):
  File "http://..../#__main__", line 3, in <module>
    import xml.dom.minidom
  File "http://../Lib/site-packages/xml/dom/minidom.py", line 951, in <module>
    class Childless:
  File "http://./Lib/site-packages/xml/dom/minidom.py", line 958, in Childless
    childNodes = EmptyNodeList()
Exception: <Javascript TypeError>: obj is undefined
```

**Comment by PierreQuentel** [comment]
Thanks for the report @mrmusicman 

The commit referenced above fixes the "BYTE ORDER MARK" issue.

Regarding the 404 errors I suggest that you take a look at the FAQ.

Anyway, the lxml package won't work because it relies on code written in C, which Brython doesn't support.
